.changeset/odd-lions-enjoy.md

```diff
@@ -0,0 +1,6 @@
+---
+"@farmfe-examples/vite-adapter-svelet": patch
+"@farmfe/core": patch
```

Comment:
The patch version of create-farm should be released.

Reply:
Thanks it's my mistake, I'll fix this now